pallets/attestation/src/lib.rs

```diff
@@ -127,7 +127,7 @@ pub mod pallet {
 		type Currency: Currency<AccountIdOf<Self>> + ReservableCurrency<AccountIdOf<Self>>;
 
 		/// The deposit that is required for storing an attestation.
-		#[pallet::constant]
+		// FIXME: Add back #[pallet::constant] once the fix is merged
```

Comment:
i would rather comment out the delegation pallet constant attribute. Attestations will be used more than delegations

Reply:
Are we assuming the same deposit amount for both? That is why we need to get rid of one of them?

Reply:
There will be different deposits. Attestations should be cheap, delegations more expensive.

Reply:
doesn't work, leave as it is.